.github/workflows/rocm_ci.yml

```diff
@@ -3,14 +3,28 @@ name: ROCM_CI
 on: 
   pull_request:
     types: [labeled, synchronize, reopened]
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'     
+        required: true
+        default: 'warning'
+  schedule:
+    - cron: "15 1 * * *"
```

Comment:
I don't think this will be able to run on the xformers CI because we don't have the `self-hosted` runners

Reply:
@tenpercent ,   can you take a look at this ?

Reply:
Yes, this should stay in rocm/xformers repo

Reply:
@danthe3rd @tenpercent 
I just simply aligned the   `rocm_ci.yml` to the same one on the main branch